app/src/main/java/de/westnordost/streetcomplete/quests/fire_hydrant_diameter/AddFireHydrantDiameterForm.kt

```diff
@@ -79,4 +117,31 @@ class AddFireHydrantDiameterForm : AbstractOsmQuestForm<FireHydrantDiameterAnswe
             .show()
         }
     }
+
+    private fun updateSuggestionsButtonVisibility() {
+        suggestionsButton.isGone = lastPickedAnswers.isEmpty() || diameterInput.intOrNull != null
+    }
+
+    private fun showSuggestionsMenu() {
+        val popup = PopupMenu(requireContext(), suggestionsButton)
+
+        for (diameter in lastPickedAnswers) {
+            popup.menu.add(diameter.toString())
+        }
+
+        popup.setOnMenuItemClickListener { item ->
+            diameterInput.setText(item.title.toString())
+            true
+        }
+        popup.show()
+    }
+}
+
+private fun getHydrantDiameterSignLayoutResId(countryCode: String): Int = when (countryCode) {
+    "DE" -> R.layout.quest_fire_hydrant_diameter_sign_de
+    "FI" -> R.layout.quest_fire_hydrant_diameter_sign_fi
+    "NL" -> R.layout.quest_fire_hydrant_diameter_sign_nl
+    "PL" -> R.layout.quest_fire_hydrant_diameter_sign_pl
+    "UK" -> R.layout.quest_fire_hydrant_diameter_sign_uk
```

Comment:
```suggestion
    "GB", "IE" -> R.layout.quest_fire_hydrant_diameter_sign_uk
```

Also, what about Belgium (BE)? They used to be defined before. It seems to have the same style as one of the others in this list, but I do not remember which.

Reply:
Ah, I thought those two files were duplicated because a country could consist of multiple MCCs. Good that we get rid of that now.

Belgium used the German sign before.